Core/HLE/sceAtrac.cpp

```diff
@@ -1964,6 +1964,8 @@ static const u16 at3plusHeaderMap[][3] = {
 	{ 0x4524, 0x0, 0x0 },
 	{ 0x02E8, 0x1, 0x0 },
 	{ 0x5C24, 0x0, 0x0 },
+	{ 0x0118, 0x1, 0x0 },
+	{ 0x2224, 0x0, 0x0 },
```

Comment:
That's... confusing.

Shouldn't it just be `{ 0x00C0, 0x1, 0x1724 },`, `{ 0x0180, 0x1, 0x2224}` etc. then?  But I guess that's not the problem of this pull request.

-[Unknown]